src/app/web-app.component.ts

```diff
@@ -113,6 +126,141 @@ export class WebAppComponent implements OnInit {
         verticalPosition: 'top'
       });
     });
-  }
 
+    // Keyboard Shortcuts Listener
+    this.unlisten = this.keyboardShortcuts.listen(
+      {
+        // Navigation Page
+        'Control.Shift.n': (event: KeyboardEvent): void => {
+          this.router.navigate(['navigation']);
+          event.preventDefault();
+        },
+
+        // Checker Inbox & Pending Tasks
+        'Control.Shift.i': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Collection Sheet
+        'Control.Shift.o': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Client
+        'Control.Shift.c': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Group
+        'Control.Shift.g': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Create Center
+        'Control.Shift.q': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Frequent Postings
+        'Control.Shift.f': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries/frequent-postings']);
+          event.preventDefault();
+        },
+
+        // Closure Entries
+        'Control.Shift.e': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Journal Entries
+        'Control.Shift.j': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting/journal-entries']);
+          event.preventDefault();
+        },
+
+        // Reports
+        'Control.Shift.r': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
+          event.preventDefault();
+        },
+
+        // Accounting
+        'Control.Shift.a': (event: KeyboardEvent): void => {
+          this.router.navigate(['accounting']);
+          event.preventDefault();
+        },
+
+        // Save/Submit Forms
+        'Control.s': (event: KeyboardEvent): void => {
+          // this.router.navigate(['navigation']);
+          console.log('Not Implemented Yet!!');
```

Comment:
There are various forms in the accounting section which have been implemented. Figure out a way so that this shortcut works there.

Reply:
This shortcut has to implemented differently for every table as each one has a different paginator. In the community app, it was implemented as a click event on the next button, but here it's not possible as the click event has to be attached to the next button which is a part of matPaginator to which we don't have access to. Let me know what should i do.

Reply:
I didn't understand. The click button already works. Are you able to figure out a way using which custom keyboard shorcuts (for next and previous) can be implemented for angular material paginator globally?

Reply:
What i was saying is that in community-app they've used a simple js script in which they grab the button HTML element of next/previous and call the click() method, which can't be implemented here also i tried but i was unable to implement shortcut for paginator as i can't access the code of the paginator library. I way i figured was to implement the shorcut individually for each for paginator, but this will lead to writing code individually for each form which will not be good according to me, what do you suggest?

Reply:
I'd suggest finding a better global way to implement this. If not possible add a relevant TODO comment here for now.